src/host/ncp_spinel.hpp

```diff
@@ -423,6 +423,16 @@ class NcpSpinel : public CliDaemon::Dependencies
      */
     void SetHostPowerState(uint8_t aState, AsyncTaskPtr aAsyncTask);
 
+#if OTBR_ENABLE_DHCP6_PD && OTBR_ENABLE_BORDER_ROUTING
+    /**
+     * This method enables/disables the DHCP6 PD on NCP.
+     *
+     * @param[in] aEnable  A boolean to enable/disable the DHCP6 PD.
+     */
+    void BorderRoutingDhcp6PdSetEnabled(bool aEnabled);
```

Comment:
Other methods in this file all use `Set<SomeFeatue>Enabled()`. 
Suggest following the same naming convention.